### PR TITLE
Adding docs around CVE-2018-18264

### PR DESCRIPTION
**Pull request by hyperbolic2346**

### CODE REVIEW

File: pages/k8s/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# CVE-2018-18264
+
+## What happened
+
+- A security vulnerability was found in the Kuberentes dashboard that affected all
```
Comment (tvansteenburgh) [inline pages/k8s/release-notes.md]: Kubernetes

File: pages/k8s/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# CVE-2018-18264
```
Comment (tvansteenburgh) [inline pages/k8s/release-notes.md]: @evilnick Do you think we should we start adding a date to these headers so it's clear when the release happened?

File: pages/k8s/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,19 @@ layout: [base, ubuntu-com]
...
+
+## What happened
+
+- A security vulnerability was found in the Kuberentes dashboard that affected all
+versions of the dashboard.
+
+A new dashboard version, v1.10.1, was released to address this vulnerability. This
+includes an important change to logging in to the dashboard. The skip button is now
```
Comment (tvansteenburgh) [inline pages/k8s/release-notes.md]: s/skip/Skip/

File: pages/k8s/release-notes.md (hunk 2 above)
Comment (evilnick) [reply pages/k8s/release-notes.md]: Yes!

Comment (tvansteenburgh) [reply pages/k8s/release-notes.md]: Yeah, but I prefer January 10, 2019 or 2019-01-10.

**Pull request merged**